lib/content-services/src/lib/document-list/components/file-auto-download/file-auto-download.component.scss

```diff
@@ -0,0 +1,17 @@
+.adf {
+    &-dialog {
```

Comment:
this is not needed if you use standard material dialog

Reply:
Agreed. After moving to mat-dialog directives and components, this scss file was not needed anymore. Removed the file and its refereneces altogether